src/components/fields/AddressField.js

```diff
@@ -77,19 +86,22 @@ const AddressField = (
     [expandAbbreviatedClipboard, onChange, validateAddress]
   );
 
+  const autofill = name || address;
+
   useEffect(() => {
-    if (name !== inputValue) {
-      setInputValue(name);
-      validateAddress(address);
+    if (!inputValue && autofill && autofill !== inputValue) {
+      setInputValue(autofill);
+      validateAddress(autofill);
     }
-  }, [address, inputValue, isValid, name, validateAddress]);
+  }, [autofill, inputValue, validateAddress]);
 
   return (
     <Row flex={1}>
       <AddressInput
         {...props}
         autoFocus={autoFocus}
         color={isValid ? colors.appleBlue : colors.dark}
+        editable={!disableInput}
```

Comment:
i like this, what do you think of changing the name to `editable` instead of `disableInput` just to have the same words